packages/tree-select/package.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "@automattic/tree-select",
+	"version": "0.1.0",
```

Comment:
Let's not do that pre-stable dance and start at `1.0.0`.

Reply:
Once we've got the lerna workflow stable, I would agree. I was going pre-stable here to give myself some room to experiment with pre-stable versions.